modules/http4s-swagger/src-jvm-ce2/Compat.scala

```diff
@@ -50,21 +55,19 @@ private[smithy4s] object Compat {
   }
 
   trait DocsCompanion extends SwaggerUiInit {
-    def apply[F[_]](
-        hasId: HasId,
+    def build[F[_]](
```

Comment:
This method should disappear plainly and simply in favour of "smart-setters" in the `PartiallyApplied` construct.